src/app/app-routing.module.ts

```diff
@@ -16,6 +16,7 @@ const routes: Routes = [
   {path: 'register', component: RegisterComponent },
   {path: 'nonprofit/:nonprofit-id', component: NonprofitPageComponent},
   {path: 'nonprofits', component: NonprofitSearchPageComponent},
```

Comment:
[NITPICK] There may be a way to combine these two routes into a single route with an optional param or something like that. Not sure though.